README.md

```diff
@@ -67,7 +67,7 @@ The following are some of the `make` targets are included:
 - manifests - generates the `manifests/` scripts
 - catalog - updates the `catalog-manifests/` if there are any updates
 - build - build the container image
-- push - pushes the container image
+- push - pushes the container image (after verifying operator's currentCSV images exist)
```

Comment:
You should add a note about the `check-operator-images` target in this list too.

Reply:
It's a dependency of `push` and we don't call out all the dependencies.  Is there a use case where someone would want to use it that didn't require a push?

Reply:
I didn't realize we omit the dependencies. Carry on. ;)